pkg/cache/cluster.go

```diff
@@ -248,7 +250,9 @@ type clusterCache struct {
 	openAPISchema               openapi.Resources
 	gvkParser                   *managedfields.GvkParser
 
-	respectRBAC int
+	respectRBAC                  int
+	listResourcesUsingWatchAPI   atomic.Int32
+	listResourcesUsingRegularAPI atomic.Int32
```

Comment:
I think maybe we shouldn't do this, because listResources gets called a lot over the lifetime of a controller run (i.e. every time the cache expires or there's a retry due to an error). I don't think it's worth it just for the test. And maybe we can find a different way to test.

Reply:
Agree that it's not ideal, I'll think how to test better.